lua/autorun/misc.lua

```diff
@@ -87,6 +87,22 @@ if SERVER then
 			end)
 		end
 	end)
+	
+	hook.Add("InitPostEntity","FindSwampJeeps",function()
+		local swamp_jeeps = ents.FindByModel("models/buggy.mdl")
+	
+		-- NOOOOO YOU CANT JUST LIMIT THE JEEPS TO ONE AREA!!!
+		-- haha jeeps go weee
+		if IsValid(swamp_jeeps) then
+			hook.Add("Tick","JeepTeleporter",function()
+				for k,v in pairs(swamp_jeeps) do
+					if IsValid(v) and v:GetLocation() != "Moon" then
```

Comment:
ent:GetLocation() isn't defined, that's only for players. Probably need to set a threshold on GetPos().z